test/features/user_can_cancel_a_new_skill_test.rb

```diff
@@ -0,0 +1,12 @@
+require_relative '../test_helper'
+
+class UserCanCancelANewSkillTest < FeatureTest
+  def test_user_can_cancel_adding_a_new_skill
+    seed_skills
```

Comment:
I like that you're trying to use helper methods, however the only problem with this one you are creating 2 skills any time you call the method #seed_skills and sometimes you only need 1 skill to do a test. We always want to create the minimum amount of data necessary to perform a test.